.gitignore

```diff
@@ -21,3 +21,5 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+node_modules
+node_modules
```

Comment:
It's a bummer that this doesn't seem to work.  I feel like I'm forgetting something obvious 🙁

Since `node_modules` is already ignored on line 4 and these extra rule sets don't seem to have an effect, they should be removed.

```suggestion
```

Reply:
Also, don't worry about this for now; let's tackle it another time.